MFVNeutralino/test/TriggerStudies/eff.py

```diff
@@ -44,43 +62,58 @@
         process.weightSeq.insert(0, process.prefiringweight * process.jmtWeightMiniAODL1Ecal)
     else:
         w = process.jmtWeightMiniAOD
-    which = 'NonPrefiringProb'
+    which = 'nonPrefiringProb'
     if 'up' in weight_l1ecal:
         which += 'Up'
     elif 'down' in weight_l1ecal:
         which += 'Down'
     w.weight_misc = True
-    w.misc_srcs = cms.VInputTag(cms.InputTag('prefiringweight', which))
+    w.misc_srcs = cms.VInputTag(cms.InputTag('prefiringweight', which, 'BasicAnalyzer'))
+
+# MET correction
+runMetCorAndUncFromMiniAOD(process,
+                           isData = not settings.is_mc,
+                           )
```

Comment:
Here (and maybe L45, and others) it could be useful to make this configurable, so that the script either can measure the MET trigger efficiency, the HT trigger efficiency, or the others.

Reply:
I think the MET correction and MET filters in the process won't influence the measurement of other trigger. As long as the require_metfilters are set to False and require_hlt is set to the corresponding trigger